src/sage/env.py

```diff
@@ -478,8 +469,8 @@ def uname_specific(name, value, alternative):
     aliases["ARB_LIBRARY"] = ARB_LIBRARY
 
     # TODO: Remove Cygwin hack by installing a suitable cblas.pc
```

Comment:
Why not remove completely?

Reply:
I don't know what's this all about. Is there a special hack in place which is only actually needed on Cygwin?

Reply:
Not any more. Removed in 1713eed49f4.